src/pip/_internal/utils/compat.py

```diff
@@ -72,7 +72,7 @@
     # backslash replacement for all versions.
     def backslashreplace_decode_fn(err):
         raw_bytes = (err.object[i] for i in range(err.start, err.end))
-        if sys.version_info[0] == 2:
+        if PY2:
```

Comment:
This check can be removed now since this function is only used in Python 2.

Reply:
Done!